cabal.project

```diff
@@ -291,44 +135,44 @@ source-repository-package
     io-sim
     strict-stm
 
+-- Temporary patch for version 0.1.0.0 (181601bc3d9e9d21a671ce01e0b481348b3ca104)
+-- in order to change io-sim from 606de33fa2f467d108fb1efb86daeb3348bf34e3
+-- to 57e888b1894829056cb00b7b5785fdf6a74c3271 to include the ReaderT fix.
```

Comment:
Again, this seems like a place where simply leaving a comment designed to be discovered exactly when it is needed seems useful. I also see less upsides here, at least to create a specific task; this "TODO" can't be tackled without the next node bump...

...but I could create a task for the next node bump and link/comment. The only problem is that we don't necessarily know for sure what version (and task title) it will be, but that seems fine anyway.

Reply:
https://input-output.atlassian.net/browse/ADP-2447